.buildkite/scripts/common.sh

```diff
@@ -6,15 +6,21 @@ WORKSPACE="$(pwd)/bin"
 TMP_FOLDER_TEMPLATE_BASE="tmp.fleet-server"
 REPO="fleet-server"
 
-add_bin_path(){
-    echo "Adding PATH to the environment variables..."
+create_workspace() {
+    if [[ ! -d "${WORKSPACE}" ]]; then
     mkdir -p ${WORKSPACE}
+    fi
```

Comment:
Maybe it can be removed the if condition.
`mkdir -p` does not fail if `WORKSPACE` folder exists. But it could be kept if you prefer.

If kept the `if` block, just a quick suggestion
```suggestion
    if [[ ! -d "${WORKSPACE}" ]]; then
        mkdir -p ${WORKSPACE}
    fi
```